flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobVertexDetailsInfo.java

```diff
@@ -122,6 +122,7 @@ public int hashCode() {
 		public static final String FIELD_NAME_END_TIME = "end-time";
 		public static final String FIELD_NAME_DURATION = "duration";
 		public static final String FIELD_NAME_METRICS = "metrics";
+		public static final String FIELD_NAME_RESOURCE_ID = "resource-id";
```

Comment:
`resource-id` is ambiguous; it's not clear what resource the ID belongs to. `taskmanager-id` seems more appropriate.